### ISSUE TITLE
`flutter:getText` —  Command not support

### ISSUE THREAD

**Issue by ichanmrg** [appium-userland/appium-flutter-driver#193]
Hi — 

I am having an error when trying to use `flutter::getText`. As per documentation, the said command is supported but it seems it is not the case on my end
![Screen Shot 2021-09-30 at 1 12 33 PM](https://user-images.githubusercontent.com/32838072/135391376-43790987-f15a-4146-8502-60a282e3d2b4.png)
Does anyone also encountered this issue?

**Comment by KazuCocoa** [comment]
I think https://github.com/appium-userland/appium-flutter-driver/blob/ce2315acf28a3b92386779ecb9e7a45d62a3eca9/example/java/src/test/java/example/appium/FlutterTest.java#L72 may help.
https://github.com/appium-userland/appium-flutter-driver#commands does not address `flutter:` command in `getText`

**Comment by ichanmrg** [comment]
> I think
> 
> https://github.com/appium-userland/appium-flutter-driver/blob/ce2315acf28a3b92386779ecb9e7a45d62a3eca9/example/java/src/test/java/example/appium/FlutterTest.java#L72
> 
> may help.
> https://github.com/appium-userland/appium-flutter-driver#commands does not address `flutter:` command in `getText`

Hi! Unfortunately, if I use the `variable.text.toString()` or `variable.getText().toString()`command, I am also encountering an error.

![Screen Shot 2021-10-01 at 4 58 14 PM](https://user-images.githubusercontent.com/32838072/135593733-b31876bc-2614-4f58-9078-6608fbe7500f.png)

**Comment by KazuCocoa** [comment]
it seems the Flutter responded as the element has many elements (?). The error is by Flutter (not by this driver). Probably you should adjust your test code to select an element or product code.

**Comment by ichanmrg** [comment]
> it seems the Flutter responded as the element has many elements (?). The error is by Flutter (not by this driver). Probably you should adjust your test code to select an element or product code.

That is my concern too after realizing it, but I tried to use `descendant` and `ancestor` functions to be more specific with the element that I want to target but it seems that there is some casting issue. I logged the said issue on a separate one.

Closing this one out and I will wait for the never-ending and gracious help of the contributors. You guys are heaven-sent!
